### ISSUE TITLE
Version 0.12.0 is broken

### ISSUE THREAD

**Issue by zenovich** [akrylysov/algnhsa#21]
```
# github.com/akrylysov/algnhsa
../../akrylysov/algnhsa/request.go:102:12: undefined: http.NewRequestWithContext
```

**Comment by zenovich** [comment]
1.12.9

**Comment by akrylysov** [comment]
Fixed in 0.12.1. Sorry for inconvenience.

**Comment by zenovich** [comment]
Thanks!
